### ISSUE TITLE
Using clipboard as default register.

### ISSUE THREAD

**Issue by f3fora** [AckslD/nvim-neoclip.lua#13]
On linux, clipboard is usually set as 
``` lua
vim.opt.clipboard = {'unnamed', 'unnamedplus'}
```
to handle also the system clipboard.

Thus, the default register becomes `+` or `*`, which eventually are updated with `"` contents.
However this plugin stores the contents  only in register `"` and they cannot be accessed by default register, because the registers `*` and `+`  are not updated.

Moreover `:Telescope neoclip *` or `+` fails with `telescope.nvim/lua/telescope/command.lua:173: attempt to call a nil value`

**Comment by AckslD** [comment]
Hi @f3fora, after #11 you can set what register you want by default by for example:
```lua
use {
  "AckslD/nvim-neoclip.lua",
  config = function()
    require('neoclip').setup({
      default_register = '+',
    })
  end,
}
```
(for more details see README)

If you want to call these explicitly without using the default you should use `Telescope neoclip plus` or `Telescope neoclip star` (I forgot to document that in the README but will add it).

Let me know if it doesn't work for you.

**Comment by f3fora** [comment]
It works.
However, copying into multiple registers at once (e.g. `", *, +`) would be a great feature. One should be able to use, for example
```lua
default_register = {'+', '*', '"'}
```
or `Telescope neoclip unnamed a`

**Comment by AckslD** [comment]
Good idea, I'll open a new issue for that :)

**Comment by AckslD** [comment]
@f3fora thinking about this more I'm not sure I fully understand the usecase for putting something in multiple registers. Could you explain how you would like to use this?

**Comment by f3fora** [comment]
On Linux `*` and  `+` are different registers, respectively middle-mouse button paste and Ctrl-V paste. One usually wants to have a common behavior. 

See `:help clipboard-x11` and [this](https://unix.stackexchange.com/questions/196274/copy-to-both-x11-clipboards-from-vim).

Moreover, with `vim.opt.clipboard = {'unnamed', 'unnamedplus'}` set, a yank action save the text in `"`, `+` and `*`. 

> @f3fora thinking about this more I'm not sure I fully understand the usecase for putting something in multiple registers. Could you explain how you would like to use this?

This is a generalization. If one can choose more registers, they can be `"`, `+`  and `*`.

**Comment by joshuarubin** [comment]
using `:Telescope neoclip plus` works for me, but setting `default_register` is not working.

I'd also like to 👍🏻 the ability to set multiple registers.

**Comment by AckslD** [comment]
Thanks for explaining! It makes sense to me :+1: I would have to figure out how to allow for variable number of arguments to the telescope call.

Seems only ones with a keyword are passed along to the picker. Ie if I do:
```
Telescope neoclip a b
```
`opts` passed to the picker does not contain `b` but if I do
```
Telescope neoclip a extra=b
```
then `opts` has the key-value pair `extra,b`.

Anyone of you know how to do this? :)

**Comment by f3fora** [comment]
You can do 
```
Telescope neoclip a,b
```
And then split the string with comma.

**Comment by AckslD** [comment]
That could work, although I suspect it might have to be
```
Telescope neoclip a b,c
```
Since the first is the "command"

**Comment by AckslD** [comment]
@f3fora @joshuarubin I added support for multiple registers in #23, feel free to check it out and let me know if it works for you
